README.md

```diff
@@ -21,19 +21,33 @@ You can install them using yum, but beware some of them are on epel and centos-r
 # yum install centos-release-scl
 # yum install llvm-toolset-7
 ```
+
+You need to create a postgres user to execute PostgreSQL.
+
+```
+# useradd postgres
```

Comment:
Although you just add postgres user, postgresqlXX-server rpm specifies various things.
Wouldn't it be better to make the settings the same as the rpm file?

```
$ rpm -qp --scripts postgresql13-server-13.1-1PGDG.rhel8.x86_64.rpm 
..(snip)..
preinstall scriptlet (using /bin/sh):
groupadd -g 26 -o -r postgres >/dev/null 2>&1 || :
useradd -M -g postgres -o -r -d /var/lib/pgsql -s /bin/bash \
        -c "PostgreSQL Server" -u 26 postgres >/dev/null 2>&1 || :
..(snip)..
```

Reply:
OK! I will modify.